### ISSUE TITLE
How you segment the brain?

### ISSUE THREAD

**Issue by omidomidi77** [bthyreau/hippodeep_pytorch#6]
Hi guys,
I see that in your new version, you produce a brain mask as well. I wonder what network/ backbone you are using for segmenting the brain, and if you mention it in your paper.

**Comment by bthyreau** [comment]
Hi,
Thanks - Indeed, Hippodeep produces a brainmask as a side effect. Additionally, the mask works on many modalities beyond T1WI.

This is based on a small custom model that i designed for the particular purpose of robust and fast brain extraction (back in 2018, GPU were less widespread among regular users). I usually don't mention the details it in papers because I don't consider brain extraction in itself as a worthwhile research problem.

The dataset was made from a variety of heterogeneous brain data found on the internet, with heavily simulated alterations for robustness. Unfortunately, I think i lost the training code when moving from Theano to Pytorch

Note that the mask is computed at lower resolution, because it was intended to solve practical needs, rather than make a "pixel-perfect" mask
